tests/unit/test_main.py

```
@@ -5,235 +5,209 @@
-
-
-@patch("markdown_lab.core.scraper.requests.Session.get")
-def test_scrape_website_general_error(mock_get, scraper):
-    mock_get.side_effect = Exception("Connection error")
-
-    with pytest.raises(Exception) as exc_info:
-        scraper.scrape_website("http://example.com")
-    assert str(exc_info.value) == "Connection error"
+    mock_request.return_value = mock_response
 
+    with pytest.raises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)